include/exadg/acoustic_conservation_equations/spatial_discretization/operators/operator.h

```diff
@@ -371,28 +362,28 @@ class Operator
       }
       else if(data.formulation == Formulation::Strong)
       {
-        pressure_m.submit_value(rhocc * (flux_momentum - um) * n, q);
-        velocity_m.submit_value(rho_inv * (flux_mass - pm) * n, q);
+        pressure_m.submit_value((flux_momentum - um) * n, q);
+        velocity_m.submit_value((flux_mass - pm) * n, q);
 
         if constexpr(weight_neighbor)
         {
           // minus signs since n⁺ = - n⁻
-          pressure_p.submit_value(-rhocc * (flux_momentum - up) * n, q);
-          velocity_p.submit_value(-rho_inv * (flux_mass - pp) * n, q);
+          pressure_p.submit_value((up - flux_momentum) * n, q);
+          velocity_p.submit_value((pp - flux_mass) * n, q);
         }
       }
       else if(data.formulation == Formulation::SkewSymmetric)
       {
-        scalar const flux_momentum_weak = rhocc * flux_momentum * n;
+        scalar const flux_momentum_weak = flux_momentum * n;
 
         pressure_m.submit_value(flux_momentum_weak, q);
-        velocity_m.submit_value(rho_inv * (flux_mass - pm) * n, q);
+        velocity_m.submit_value((flux_mass - pm) * n, q);
 
         if constexpr(weight_neighbor)
         {
           // minus signs since n⁺ = - n⁻
           pressure_p.submit_value(-flux_momentum_weak, q);
-          velocity_p.submit_value(-rho_inv * (flux_mass - pp) * n, q);
+          velocity_p.submit_value((pp - flux_mass) * n, q);
```

Comment:
Is there docu in the spatial operator that "velocity" describes `rho*u`? Maybe, writing `rho_u` explicitly here for the variable names would also be clearer here.